Source/WebCore/contentextensions/ContentExtensionParser.cpp

```diff
@@ -310,11 +310,14 @@ static std::optional<Expected<ContentExtensionRule, std::error_code>> loadRuleId
     if (!identifier.has_value())
         return makeUnexpected(ContentExtensionError::JSONInvalidRuleIdentifier);
 
+    auto rulesetIdentifierValue = ruleObject.getValue("_rulesetIdentifier"_s);
+    auto rulesetIdentifier = rulesetIdentifierValue ? rulesetIdentifierValue->asString() : nullString();
```

Comment:
```suggestion
    auto rulesetIdentifier = ruleObject.getString("_rulesetIdentifier"_s);;
```
^ would do the same thing, I think